docs/sphinx/how_to/magma_install/magma_install.md

```diff
@@ -8,7 +8,8 @@ Deep Learning applications. Figure 5 provides the sequential flow for the use of
 each framework. Refer to the ROCm Compatible Frameworks Release Notes for each
 framework's most current release notes at
 /bundle/ROCm-Compatible-Frameworks-Release-Notes/page/Framework_Release_Notes.html.
-![Figure 5][/figures/image.005.png]
+
+![Figure 5](/figures/image.005.png)

```

Comment:
@Ehud-Sharlin Your directory is spelled with capital Figure, please change to lowercase. It is case sensitive.
Also, link should be (figures/image.005.png)

Reply:
The vscode markdown preview shows Figure 5, while the readthedocs rendered version doesn't... I'll fix this tomorrow!

Reply:
Fixed

Reply:
Thanks Saad!
A quick question: how should I handle in Markdown the code snippets? (e.g. "docker pull rocm/pytorch:latest"). 
Thanks
(p.s. I will to go offline for a few hours but will get back to this later today)

Reply:
``` test ```